### ISSUE TITLE
app.wireflow.com not working on Firefox 

### ISSUE THREAD

**Issue by tamielbr** [vanila-io/wireflow#38]
Running on Mint 19.3, Firefox 77.0.1 (64-bit). 

When trying to use https://app.wireflow.co/, one cannot drag any items from the left to the main canvas. On the same setup, works fine on Chrome, can drag items with no issues.

Short video (6 sec) - https://streamable.com/qcm0ey

**Comment by tamielbr** [comment]
This is a bit weird. But I can make it work with the _right_ mouse button! Not sure what is going on (not expected), but on Chrome it works as expected with left button.

**Comment by shtefcs** [comment]
You are right, its buggy and DnD doesn't work from FF. Thanks for reporting, we will take care of it on next release 👍
